internal/builders/docker/commands.go

```diff
@@ -87,20 +92,24 @@ func BuildCmd(check func(error)) *cobra.Command {
 			// Remove any temporary files that were generated during the setup.
 			defer db.RepoInfo.Cleanup()
 
-			artifacts, err := db.BuildArtifacts()
+			// Build artifacts and write them to the output folder.
+			artifacts, err := db.BuildArtifacts(absoluteOutputFolder)
```

Comment:
not sure where we verify the `absoluteOutputFolder`. This input is always trusted and provided by the reusable workflow e control? Maybe add a comment. Also you could verify that it starts with `/tmp/`.

Reply:
Yeah, it is always provided by the reusable workflow to internally pass the built artifacts. 
The only verification that needs to happen is that it doesn't overwrite any existing files I think, because it may not be in the current working directory. That check happens when we write a path.

> Maybe add a comment. Also you could verify that it starts with /tmp/.

Yeah, I think this might be the better case. It was really hard to think of exactly what verification was needed here. But really I need a tmp out.